text/0000-ppv.md

```diff
@@ -0,0 +1,1471 @@
+
+/// Instantiates a new vector with the values of the `slice`.
+///
+/// # Precondition
+///
+/// If `slice.len() < Self::lanes()` or `&slice[0]` is not aligned
+/// to an `align_of::<Self>()` boundary, the behavior is undefined.
+pub unsafe fn load_aligned_unchecked(slice: &[element_type]) -> Self;
+
+/// Instantiates a new vector with the values of the `slice`.
+///
+/// # Precondition
+///
+/// If `slice.len() < Self::lanes()` the behavior is undefined.
+pub unsafe fn load_unaligned_unchecked(slice: &[element_type]) -> Self;
+}
+```
+
+#### Vertical arithmetic operations
+
+Vertical (lane-wise) arithmetic operations are provided by the following trait
+implementations:
+
+* All signed integer, unsigned integer, and floating point vector types implement:
+
+    * `{Add,Sub,Mul,Div,Rem}<RHS=Self,Output=Self>`
+    * `{Add,Sub,Mul,Div,Rem}Assign<RHS=Self>`
+
+* All signed and unsigned integer vectors also implement:
+
+    * `{Shl,Shr}<RHS=Self,Output=Self>`, `{Shl,Shr}Assign<RHS=Self>`: vertical
+      (lane-wise) bit-shift operations.
+
+##### Integer vector semantics
+
+The behavior of these operations for integer vectors is the same as that of the
+scalar integer types. That is: `panic!` on both overflow and division by zero in
+debug mode.
+  
+##### Floating-point semantics
+
+The behavior of these operations for floating-point numbers is the same as that
+of the scalar floating point types, that is, `+-INFINITY` on overflow, `NaN` on
+division by zero, etc.
+
+#### Wrapping arithmetic operations
+
+All signed and unsigned integer vector types implement the whole set of `pub fn
+wrapping_{add,sub,mul,div,rem}(self, Self) -> Self` methods which, on overflow,
+produce the correct mathematical result modulo `2^n`.
+
+The `div` and `rem` method `panic!` on division by zero in debug mode.
+
+#### Unsafe wrapping arithmetic operations
+
+All signed and unsigned integer vectors implement
+`pub unsafe fn wrapping_{div,rem}_unchecked(self, Self) -> Self`
+methods which, on overflow, produce the correct mathematical result modulo `2^n`.
+
+If any of the vector elements is divided by zero the behavior is undefined.
+
+#### Binary `min`/`max` vertical operations
+
+All portable signed integer, unsigned integer, and floating-point vectors
+implement the following methods:
+
+```rust
+impl {element_type}{lane_width}x{number_of_lanes} {
+/// Lane-wise `min`.
+///
+/// Returns a vector whose lanes contain the smallest 
+/// element of the corresponding lane of `self` and `other`.
+pub fn min(self, other: Self) -> Self;
+
+/// Lane-wise `max`.
+///
+/// Returns a vector whose lanes contain the largest 
+/// element of the corresponding lane of `self` and `other`.
+pub fn max(self, other: Self) -> Self;
+}
+```
+
+##### Floating-point semantics
+
+The floating-point semantics follow the semantics of `min` and `max` for the
+scalar `f32` and `f64` types. That is:
+
+If either operand is a `NaN`, returns the other non-NaN operand. Returns `NaN`
```

Comment:
@gnzlbg This section also unnecessarily describes behavior that matches the scalar case.